test/View/RouteNotFoundStrategyTest.php

```diff
@@ -16,9 +16,12 @@
 use Zend\Mvc\MvcEvent;
 use Zend\Mvc\View\Http\RouteNotFoundStrategy;
 use Zend\View\Model\ViewModel;
+use ZendTest\Mvc\EventManagerIntrospectionTrait;
 
 class RouteNotFoundStrategyTest extends TestCase
 {
+    use EventManagerIntrospectionTrait;
```

Comment:
Why? What rationale can you provide?
On Oct 1, 2015 6:46 AM, "Marco Pivetta" <notifications@github.com> wrote:

> In test/View/RouteNotFoundStrategyTest.php
> <https://github.com/zendframework/zend-mvc/pull/31#discussion_r40904278>:
>
> >
> >  class RouteNotFoundStrategyTest extends TestCase
> >  {
> > +    use EventManagerIntrospectionTrait;
>
> After reading through the tests, I'm more convinced that the trait should
> just be a static class
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-mvc/pull/31/files#r40904278>.
>